client/web/src/components/users/Signin/SignIn.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
+import Avatar from '@material-ui/core/Avatar'
+import Button from '@material-ui/core/Button'
+import CssBaseline from '@material-ui/core/CssBaseline'
+import TextField from '@material-ui/core/TextField'
+import Grid from '@material-ui/core/Grid'
+import { Link } from '@material-ui/core'
+import LockOutlinedIcon from '@material-ui/icons/LockOutlined'
+import Typography from '@material-ui/core/Typography'
+import { makeStyles } from '@material-ui/core/styles'
+import Container from '@material-ui/core/Container'
+
+import { Link as RouterLink, useRouteMatch, useHistory } from 'react-router-dom'
+import { useDispatch } from 'react-redux'
+import { useTranslation } from 'react-i18next'
+
+import { actionTypes } from '../../../features/user'
+
+import style from './SignIn.style'
+
+const useStyles = makeStyles(style)
+
+export const SignIn: React.FC = () => {
+  const classes = useStyles()
+  const location = useRouteMatch()
+  const dispatch = useDispatch()
+  const history = useHistory()
+  const { t } = useTranslation()
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault()
+    // Call the API to login & returns informations about logged user
+    dispatch({
+      type: actionTypes.SET_CURRENT_USER,
+      user: { lastname: 'damien' },
```

Comment:
Is it really wanted ? If yes why ? ^^

Reply:
yes we must store the current user in store, but we will have to keep some user data like firstname, lastname, auth token (for api calls)

For the moment it is a fake